rust/crates/kafka-es-indexer/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "kafka-es-indexer"
```

Comment:
Sorry, I known changing a project name can be a pain, but may I suggest a new name?

I don't think kafka-es-indexer is specific enough. It tells me the module is meant to index something from kafka into es, but I doesn't tell me why, and on the context of a render farm management system, people may assume kafka is a part of the broad rendering solution, when it is actually only being used for monitoring.

Something like **monitoring-consumer** or even **monitoring-indexer**. Anything that gives a sense this is a part of the monitoring stack.